### PR TITLE
Trial adding ormolu

### PR DESCRIPTION
**Pull request by Cmdv**
Seems slightly different in places nice in others, see what people think :)

### CODE REVIEW

File: hasktorch/src/Torch/Typed/NN/Recurrent/LSTM.hs [hunk 1]
```diff
@@ -189,117 +197,131 @@ instance A.Parameterized (LSTMLayer inputSize hiddenSize directionality dtype de
...
+data
+  LSTMLayerStackSpec
+    (inputSize :: Nat)
+    (hiddenSize :: Nat)
+    (numLayers :: Nat)
+    (directionality :: RNNDirectionality)
+    (dtype :: D.DType)
+    (device :: (D.DeviceType, Nat))
+  = LSTMLayerStackSpec
+  deriving (Show, Eq)
```
Comment (junjihashimoto) [inline hasktorch/src/Torch/Typed/NN/Recurrent/LSTM.hs:221]: https://github.com/hasktorch/hasktorch/pull/438
Looks good including here.

**Comment by Cmdv** [comment]
@tscholak sorry totally miss read what you meant!!

**Review by junjihashimoto — APPROVED** [review]

**Comment by Cmdv** [comment]
not too sure why this would be failing with jupyterlab:
```
cachix use iohk
nix-shell nix/jupyter-shell.nix -j 1 --option sandbox false --command echo
```
and this with linux:

```
stack --nix build

The following directories may now contain files, but won't be used by stack:
         - /home/runner/.stack/programs/x86_64-linux-nix/ghc-8.8.3.temp/
         - /home/runner/.stack/programs/x86_64-linux-nix/ghc-8.8.3/
```

I've been having a bit of a nightmare when it comes to nix & stack but that's been on MacOs!

**Comment by tscholak** [comment]
Don't worry about Jupyter and stack-nix for this PR. they have been broken for some time :/

**Pull request merged**